quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/commons/QuarkusProjectInfo.java

```diff
@@ -64,6 +64,22 @@ public void setClasspathKind(ClasspathKind classpathKind) {
 		return properties;
 	}
 
+	/**
+	 * Returns a <code>ExtendedConfigDescriptionBuildItem</code> representing the
+	 * <code>propertyName</code> property
+	 * @param propertyName the property name of interest
+	 * @return <code>ExtendedConfigDescriptionBuildItem</code> representing the
+	 * <code>propertyName</code> property
+	 */
+	public ExtendedConfigDescriptionBuildItem getPropertyFromName(String propertyName) {

```

Comment:
Remove this method to use `QuarkusPropertiesUtils#getProperty(String propertyName, QuarkusProjectInfo info)` which manages search of mapped property.